tools/build_api.py

```diff
@@ -237,7 +268,7 @@ def build_project(src_path, build_path, target, toolchain_name,
             add_result_to_report(report, cur_result)
 
         # Let Exception propagate
-        raise e
+        raise
```

Comment:
https://docs.python.org/3/reference/simple_stmts.html#raise

> If no expressions are present, raise re-raises the last exception that was active in the current scope.

Propagating the exception carries the backtrace forward and helps significantly when debuggin the tools.